.github/workflows/ci.yml

```
@@ -13,11 +13,15 @@ env:
   NXF_ANSI_LOG: false
   NXF_SINGULARITY_CACHEDIR: ${{ github.workspace }}/.singularity
   NXF_SINGULARITY_LIBRARYDIR: ${{ github.workspace }}/.singularity
+  NFTEST_VER: "0.9.2"
 
 concurrency:
   group: "${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}"
   cancel-in-progress: true
 
+permissions:
+  checks: write
+
 jobs:
   test:
     name: "Run pipeline with test data (${{ matrix.NXF_VER }} | ${{ matrix.test_name }} | ${{ matrix.profile }})"
```

To see which nf-test `tag` is run in each workflow.

```suggestion
    name: "${{ matrix.tags }} | ${{ matrix.profile }} | NF-${{ matrix.NXF_VER }}"
```